### PR TITLE
Add `outputFormat` option to compile to function body

### PR DESCRIPTION
**Pull request by wooorm**
This exposes the currently internal `_contain` option in the public interface,
which is used by `evaluate`, so that users can depend on it too.

Related to GH-23

### CODE REVIEW

File: test/esm-loader.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ test('xdm (ESM loader)', async function (t) {
     Content = await import('./context/esm-loader.mdx')
   } catch (error) {
     if (error.code === 'ERR_UNKNOWN_FILE_EXTENSION') {
+      await fs.unlink(path.join(base, 'esm-loader.mdx'))
```
Comment (ChristianMurphy) [inline test/esm-loader.js:22]: this change appears to be unrelated? What does it do?

**Review by ChristianMurphy — APPROVED** [review]

File: test/esm-loader.js (hunk 1 above)
Comment (wooorm) [reply test/esm-loader.js:22]: Yeah, was a small fix to the previous commit (on main) that I ran into again.
The esm loader requires args passed to node. This error is thrown without them. I typically do 'node some-test.js' to run a subset of the tests, where I forget that that arg is needed. And this line ensures the tests are cleaned up in that case!

Comment (ChristianMurphy) [reply test/esm-loader.js:22]: gotcha
